src/js/index.js

```diff
@@ -262,14 +262,14 @@ export default class bulmaCalendar extends EventEmitter {
    *                                                  *
    ****************************************************/
   onSelectDateTimePicker(e) {
-    this.refresh();
-    this.save();
-    if (e.type === 'select' && this.options.closeOnSelect && this.options.displayMode !== 'inline') {
+    this.refresh(); 
+    if (e.type === 'select' && this.options.closeOnSelect === true && this.options.displayMode !== 'inline') {
+      this.save();
       this.hide();
+      this.emit(e.type, this);
     }
```

Comment:
Can you explain why this change was made? Moving the `save()` and `emit()` calls inside the conditional prevents the `select` event from firing in most cases.